test/glance_test.gleam

```diff
@@ -3487,3 +3488,146 @@ pub fn main() {
     ),
   ])
 }
+
```

Comment:
Maybe I'd add a couple of tests to check the usage of punned fields alongside labelled and non-labelled ones, just to make sure:
`call(field, other, label:, non_punned: a, punned:)`

Reply:
Good idea, I'll add some!